include/vulkan/utility/vul_dispatch_table.h

```diff
@@ -62,31 +63,31 @@ typedef struct VulInstanceDispatchTable_ {
     PFN_vkCreateDisplayPlaneSurfaceKHR CreateDisplayPlaneSurfaceKHR;
 #ifdef VK_USE_PLATFORM_XLIB_KHR
     PFN_vkCreateXlibSurfaceKHR CreateXlibSurfaceKHR;
-#endif  // VK_USE_PLATFORM_XLIB_KHR
+#endif //VK_USE_PLATFORM_XLIB_KHR
```

Comment:
Did the file accidentally get run with clang-format before being checked in?

Seems like a reason to fix the code-gen, but for now a big change here is fine.

Reply:
I'd rather avoid the diff in this PR

Reply:
Yes it looks like the following commit had clang-format applied before checking in: https://github.com/KhronosGroup/Vulkan-Utility-Libraries/commit/2bd0e0d43884f1d655d66c2195539834a113f850

I didn't run clang-format on this PR so all those formatting changes got reverted

Reply:
I'm going ahead and working on a fix so that its clang-format compliant.